src/admin-views/attendees.php

```diff
@@ -161,6 +161,14 @@
 					do_action( 'tribe_events_tickets_attendees_totals_bottom', $event_id );
 					?>
 				</div>
+				<?php 
+				/**
+				 * Fires after the last column so that "extra" content can be displayed.
+				 *
+				 * @param int $event_id
+				 */
+				do_action( 'tec_events_tickets_attendees_event_summary_table_extra', $event_id ); 
```

Comment:
The prefix should be `tec_tickets`. We need to add a comment to the `$event_id` param and add a `@since` as we're introducing this action.